template.tpl

```diff
@@ -0,0 +1,417 @@
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "SELECT",
+    "name": "tagType",
+    "displayName": "Tag Type",
+    "selectItems": [
+      {
+        "value": "lp",
+        "displayValue": "Landing Page"
+      },
+      {
+        "value": "conv",
+        "displayValue": "Conversion"
+      }
+    ],
+    "simpleValueType": true,
+    "help": "If you set the tag type to \u003cstrong\u003eLanding Page\u003c/strong\u003e, the tag will collect the \u003cstrong\u003edooID\u003c/strong\u003e from the URL and store it in a cookie named \u003cstrong\u003e__doo_id\u003c/strong\u003e. If you set the tag type to \u003cstrong\u003eConversion\u003c/strong\u003e, the tag will fire the conversion pixel to dooshop.",
+    "defaultValue": "lp",
+    "alwaysInSummary": false
+  },
+  {
+    "type": "TEXT",
+    "name": "shopId",
+    "displayName": "Shop ID",
+    "simpleValueType": true,
+    "help": "This value is sent to you in the set-up email.",
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "enablingConditions": [
+      {
+        "paramName": "tagType",
+        "paramValue": "conv",
+        "type": "EQUALS"
+      }
+    ]
+  },
+  {
+    "type": "GROUP",
+    "name": "orderParams",
+    "displayName": "Order Details",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "TEXT",
+        "name": "orderId",
+        "displayName": "Order ID",
+        "simpleValueType": true,
+        "help": "Provide a string that uniquely identifies the order, such as a transaction/order ID."
+      },
+      {
+        "type": "TEXT",
+        "name": "orderValue",
+        "displayName": "Order Value (excl. taxes)",
+        "simpleValueType": true
+      },
+      {
+        "type": "TEXT",
+        "name": "currency",
+        "displayName": "Currency Code",
+        "simpleValueType": true,
+        "help": "Provide the currency code of the order in ISO 4217 format.",
+        "defaultValue": "EUR"
+      }
+    ],
+    "enablingConditions": [
+      {
+        "paramName": "tagType",
+        "paramValue": "conv",
+        "type": "EQUALS"
+      }
+    ]
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+const getCookieValues = require('getCookieValues');
+const setCookie = require('setCookie');
+const sendPixel = require('sendPixel');
+const getQueryParameters = require('getQueryParameters');
+const getTimestamp = require('getTimestamp');
+const enc = require('encodeUriComponent');
+const Math = require('Math');
+const log = require('logToConsole');
+
+const cookieName = '__doo_id';
+const urlBase = 'https://one.do.dmodule1.com/ts/';
+
+const time = Math.round(getTimestamp() / 1000);
+
+if (data.tagType === 'lp') {
+  const dooid = getQueryParameters('dooid');
+  if (dooid) {
+    setCookie(cookieName, dooid, {
+      domain: 'auto',
+      path: '/',
+      'max-age': 60*60*24*30,
+      secure: true,
+      samesite: 'None'
+    });
+  }
+  data.gtmOnSuccess();
+} else {
```

Comment:
Could you change this to` } else if (data.tagType === 'conv') { `
In case we want to add more paramters of pages.

Reply:
Fixed.